readme.md

```diff
@@ -897,6 +897,7 @@
 - [Computational Geometry](https://github.com/atkirtland/awesome-computational-geometry#readme) - Computational approaches for problems in geometry.
 - [OpenStreetMap](https://github.com/osmlab/awesome-openstreetmap#readme) - An open data mapping project utilized by many apps and devices.
 - [Computational Biology](https://github.com/inoue0426/awesome-computational-biology#readme) - Computational approaches applied to problems in biology.
+- [AI Collection](https://github.com/ai-collection/ai-collection) - A Collection of Awesome Generative AI Applications
```

Comment:
The link should include readme like this: "https://github.com/ai-collection/ai-collection#readme".
Also, the entry must end with a period and not a trailing whitespace.

Reply:
You should also change the license to something else such as CC0 rather than the MIT license as this one is not accepted.